recipes/libpng/all/conanfile.py

```diff
@@ -1,7 +1,10 @@
-from conans import ConanFile, CMake, tools
 import os
+from conan import ConanFile
+from conans import tools, CMake
+from conan.tools.files import apply_conandata_patches
+from conan.tools.build.cross_building import cross_building
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.45.0"
```

Comment:
```suggestion
required_conan_version = ">=1.47.0"
```
new tools requires 1.47.0